tests/unit/core/components/TestServerHtml.js

```diff
@@ -112,8 +120,8 @@ describe(__filename, () => {
     const root = render();
     const js = root.find('script');
 
-    expect(js.at(1)).toHaveProp('integrity', 'sha512-disco-js');
-    expect(js.at(1)).toHaveProp('crossOrigin', 'anonymous');
+    expect(js.at(2)).toHaveProp('integrity', 'sha512-disco-js');
+    expect(js.at(2)).toHaveProp('crossOrigin', 'anonymous');
```

Comment:
nit: same here..

Reply:
There is now a script injected before those scripts. It is the script tag for the JSON-LD data.